spec/index.bs

```diff
@@ -1879,94 +1894,98 @@ A {{GPUBindGroup}} object has the following internal slots:
-
-<dfn>bind group device validation</dfn>: The {{GPUDevice}} must not be lost.
-
-<dfn>texture view binding validation</dfn>: Let |view| be
-    |bindingDescriptor|.{{GPUBindGroupEntry/resource}}, a {{GPUTextureView}}.
-    This |layoutBinding| must be compatible with this |view|. This requires:
-    1. Its |layoutBinding|.{{GPUBindGroupLayoutEntry/viewDimension}} must equal |view|'s
-        {{GPUTextureViewDescriptor/dimension}}.
-    1. Its |layoutBinding|.{{GPUBindGroupLayoutEntry/textureComponentType}} must be compatible
-        with |view|'s {{GPUTextureViewDescriptor/format}}.
-    1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/multisampled}} is `true`, |view|'s texture's
-        {{GPUTextureDescriptor/sampleCount}} must be greater than 1.
-        Otherwise, if |bindingDescriptor|.{{GPUBindGroupLayoutEntry/multisampled}} is `false`,
-        |view|'s texture's {{GPUTextureDescriptor/sampleCount}} must be 1.
-    1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is
-        {{GPUBindingType/"sampled-texture"}}, |view|'s texture's {{GPUTextureDescriptor/usage}}
-        must include {{GPUTextureUsage/SAMPLED}}. Each texture [=subresource=] seen by |view|
-        is added to {{GPUBindGroup/[[usedTextures]]}} with {{GPUTextureUsage/SAMPLED}} flag.
-    1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is
-        {{GPUBindingType/"readonly-storage-texture"}} or {{GPUBindingType/"writeonly-storage-texture"}},
-        |view|'s texture's {{GPUTextureDescriptor/usage}} must include {{GPUTextureUsage/STORAGE}}.
-        Each texture [=subresource=] seen by |view|
-        is added to {{GPUBindGroup/[[usedTextures]]}} with {{GPUTextureUsage/STORAGE}} flag.
-
-<dfn>buffer binding validation</dfn>: Let |bufferBinding| be
-    |bindingDescriptor|.{{GPUBindGroupEntry/resource}}, a {{GPUBufferBinding}}.
-    This |layoutBinding| must be compatible with this |bufferBinding|. This requires:
-    1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"uniform-buffer"}},
-        the |bufferBinding|.{{GPUBufferBinding/buffer}}'s {{GPUBufferDescriptor/usage}} must include
-        {{GPUBufferUsage/UNIFORM}}. The buffer is added to the {{GPUBindGroup/[[usedBuffers]]}} map
-        with {{GPUBufferUsage/UNIFORM}} flag.
-    1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"storage-buffer"}}
-        or {{GPUBindingType/"readonly-storage-buffer"}}, the
-        |bufferBinding|.{{GPUBufferBinding/buffer}}'s {{GPUBufferDescriptor/usage}} must include
-        {{GPUBufferUsage/STORAGE}}. The buffer is added to the {{GPUBindGroup/[[usedBuffers]]}} map
-        with {{GPUBufferUsage/STORAGE}} flag.
-    1. The bound part designated by |bufferBinding|.{{GPUBufferBinding/offset}} and
-        |bufferBinding|.{{GPUBufferBinding/size}} must reside inside the buffer.
-</dl>
+<div algorithm="GPUDevice.createBindGroup">
+    **Arguments:**
+        - {{GPUBindGroupDescriptor}} |descriptor|
```

Comment:
Later in the PR you are changing '-' to '*' for the lists.
Is there a prefered markup?

Reply:
Oops, I didn't think about it. The point of the other change was just to change the indentation.

I don't think we need to prefer `*` vs `-`, though it would be good to have consistent indentation (I don't know what the least error prone indentation is, wrt accidentally including A in B):

```
  - X
  - Y
  - B
A
```

Reply:
It looks like bikeshed's "native" indentation is 2 spaces, so two spaces in front of lists (either numbered or bulleted) is enough to avoid that happening. Also it produces a compile error anyway.